### ISSUE TITLE
Handle commaseparated uid list for pages.rootPid

### ISSUE THREAD

**Issue by ayacoo** [Yoast/Yoast-SEO-for-TYPO3#230]
## Is your feature request related to a problem? Please describe.
We had a this nav structure:

- Home (1)
  - Separator (2)
    - Nav point A (3)
    - Nav point B (4)
  - Separator (5)
    - Nav point C (6)
    - Nav point D (7)

Because of the separator we can't set rootPid = 1. Our solution would be rootPid = 2,5 
Yoast can not handle this right now.

## Describe the solution you'd like

In v3 we can change the code simply:
```
$rootPids = GeneralUtility::intExplode(',',$sitemapSettings['rootPid']) ?: [$tsfe->id];
$where = $sitemapSettings['additionalWhere'] ?:'';

                foreach ($rootPids as $rootPid) {
                    $docs[] = $tsfe->sys_page->getPage($rootPid);
                    $docs = array_filter(
                        $this->getSubPages($rootPid, $docs, $where),
                        '\YoastSeoForTypo3\YoastSeo\UserFunc\SitemapXml::filterNoIndexPages'
                    );
                }
```
Technical info

- TYPO3 version: 8.7.20
- Yoast SEO version: 3.0.4

PR will follow if feature request is fine..

**Comment by ayacoo** [comment]
Better solution: Move rootPid option higher and so we can use it for pages and "other" plugins.

**Comment by ayacoo** [comment]
@RinyVT approved :-)

**Comment by haassie** [comment]
Fixed by #236
